.env

```
@@ -1 +1,2 @@
-REACT_APP_RAPID_API_KEY=KJwZZIJSFimshuivMSVGaiYzkRomp15f2vKjsnK4bKzuUzVLzA
\ No newline at end of file
+REACT_APP_RAPID_API_KEY=KJwZZIJSFimshuivMSVGaiYzkRomp15f2vKjsnK4bKzuUzVLzA
+REACT_APP_YOUTUBE_API_KEY=AIzaSyD4WKhhYFE1YnlNCfPvSrmeNRdHaMnD8wI
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)